### PR TITLE
Backport: #4147 and #4135 to 5.4

### PR DESCRIPTION
**Pull request by dedemorton**
Cherry-picks #4147 and #4135 into 5.4.

@ruflin The changes that you made in #3559 never got into the 5.4 branch for some reason, but I saw them when I resolved a merge conflict. Let me know if this statement is no longer true, and I will remove it: "The docker module is currently not tested on Windows." Otherwise, I'll leave it in.

Please feel free to merge this if it looks OK.

### CODE REVIEW

**Comment by ruflin** [comment]
@dedemorton You will need to run `make update` for this build to go green.

File: metricbeat/docs/modules/docker.asciidoc [hunk 1]
```diff
@@ -10,6 +10,15 @@ beta[]
 This module fetches metrics from https://www.docker.com/[Docker] containers.
 
 
+[float]
```
Comment (ruflin) [inline metricbeat/docs/modules/docker.asciidoc]: These lines must go into the docker module asciidoc in the _meta directory.

**Comment by dedemorton** [comment]
@ruflin I ran `make update` and pushed the change. I'm still seeing build errors, but I don't think they are related to the docs. Can you verify?

**Comment by dedemorton** [comment]
Jenkins, test it

**Pull request merged**